modules/aws/aws-infra/README.md

```diff
@@ -0,0 +1,422 @@
+# AWS Infrastructure Module for Databricks
+
+A comprehensive, production-ready AWS infrastructure module that provides all necessary resources for Databricks workloads using official AWS Terraform modules and best practices.
+
+## Overview
+
+This module creates a complete AWS infrastructure foundation optimized for Databricks, featuring:
+
+- **🔧 Simplified Configuration**: Uses official `terraform-aws-modules/vpc` for networking
+- **🔒 Secure Storage**: S3 buckets with encryption for workspace and Unity Catalog
+- **👤 IAM Integration**: Cross-account and Unity Catalog roles with Databricks-generated policies
```

Comment:
How configurable are policies? For cross-account we have different policy types: restricted/managed/...